starfyre/compiler.py

```diff
@@ -191,11 +191,11 @@ def transpile_to_python(
 
     final_python_lines.append(main_content)
 
-    file_name = output_file_name.split("/")[-1]                 #getting the file itself "without the path"
-    output_file_name = project_dir / "build" / "pages" /file_name
+    output_file_name = project_dir / "build" / output_file_name
```

Comment:
I changed these lines to ensure that we don't put files that don't belong in the `build/pages` directory there.

Reply:
nicee